00-open-a-shapefile.Rmd

```diff
@@ -318,17 +332,16 @@ span 2 lines, use `\n` where you'd like the line break.
 
 ```{r plot-multiple-shapefiles }
 #Plot multiple shapefiles
-
-plot(aoiBoundary, col = "purple", main="Harvard Forest\nStudy Area")
-plot(lines, add = TRUE)
+plot(aoiBoundary_HARV, col = "lightgreen", 
+     main="NEON Harvard Forest\nField Site")
+plot(lines_HARV, add = TRUE)
 
 #use the pch element to adjust the symbology of the points
-plot(point, add  = TRUE, pch = 19, col = "red")
+plot(point_HARV, add  = TRUE, pch = 19, col = "purple")
 ```
 
+Bonus: For more on parameter options in base `R` `plot()`, check out these
+resources:
 
-Bonus: For more on parameter options in base r plot() check out these resources:
-
-* <a href="http://www.statmethods.net/advgraphs/parameters.html" target="_blank">Parameter methods in R.</a>
-* <a href="https://codeyarns.files.wordpress.com/2011/07/20110729-vim-named-colors.png?w=700" target="_blank">Color names in R</a>
-
+* <a href="http://www.statmethods.net/advgraphs/parameters.html" target="_blank">Parameter methods in `R`.</a>
+* <a href="https://codeyarns.files.wordpress.com/2011/07/20110729-vim-named-colors.png?w=700" target="_blank">Color names in `R`</a>
```

Comment:
cool - so it looks like we are missing a wrap up challenge for this lesson? Maybe this becomes an issue?